FirebaseSharedSwift/Sources/RCValueDecoding.swift

```diff
@@ -0,0 +1,23 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Foundation
+
+public protocol RCValueDecoding {
```

Comment:
Nit: can you document this?

Reply:
Done. PTAL.  what do you think about the naming? Should it be FirebaseRemoteConfigValueDecoding?

Reply:
imo: it's better to [avoid abbreviations when naming the Swift API](https://www.swift.org/documentation/api-design-guidelines/#use-terminology-well). `FirebaseRemoteConfigValueDecoding` looks better for this.

Reply:
Done in 522f2fc29